jerry-core/ecma/builtin-objects/ecma-builtin-json.c

```diff
@@ -1252,16 +1252,30 @@ ecma_builtin_json_serialize_property (ecma_json_stringify_context_t *context_p,
     return value;
   }
 
+  bool value_is_big_int = false;
```

Comment:
There should be no reason for this variable after the other changes, just inline the bigint check.